toolkit/tools/pkg/imagecustomizerlib/shrinkfilesystems.go

```diff
@@ -143,12 +159,18 @@ func getFilesystemSizeInSectors(resize2fsOutput string, imageLoopDevice string)
 }
 
 // Get the new partition end in sectors.
-func getNewPartitionEndInSectors(resize2fsOutput string, startSector string, imageLoopDevice string) (endInSectors string, err error) {
-	filesystemSizeInSectors, err := getFilesystemSizeInSectors(resize2fsOutput, imageLoopDevice)
+func getNewPartitionEndInSectors(resize2fsStdout string, resize2fsStderr string, startSector string,
+	imageLoopDevice string,
+) (endInSectors string, err error) {
+	filesystemSizeInSectors, err := getFilesystemSizeInSectors(resize2fsStdout, resize2fsStderr, imageLoopDevice)
 	if err != nil {
 		return "", fmt.Errorf("failed to get filesystem size:\n%w", err)
 	}
 
+	if filesystemSizeInSectors < 0 {
```

Comment:
Suggestion to make the intention of this part clearer: we could add a named constant like `noResizeHappened = -1` and convert the check to `filesystemSizeInSectors == noResizeHappened`. Right now a negative number of sectors and an empty string as result adds extra burden on a dev to check how this function is used and figure out the meaning behind this check and returned values.

Alternatively or in addition to that, we could use a well-named constant for the returned string to achieve the same goal.

Reply:
I've added documentation to the functions to call out this behavior and added some extra comments.